### PR TITLE
Fetch base fees from service - Closes #3081

### LINKED ISSUE #3081: Fetch base fees from Lisk Service
### Description
The current implementation of `getTransactionBaseFees` mocks the results with realistic values. We will refactor this function to fetch the base fees from Lisk Service when the endpoint is ready.

### PR DESCRIPTION
**Pull request by isalga**
### What was the problem?
This PR resolves #3081 
### How was it solved?
- Use API endpoint
- Disable fee options when base fees are equal

### How was it tested?
Update unit tests

### CODE REVIEW

File: src/components/screens/settings/index.js [hunk 1]
```diff
@@ -6,17 +6,23 @@ import { settingsUpdated } from '../../../actions/settings';
...
 });
 
 const mapDispatchToProps = {
   timerReset,
   networkSet,
   settingsUpdated,
+  serviceUrlSet: url => ({
+    data: url,
+    type: actionTypes.serviceUrlSet,
+  }),
```
Comment (reyraa) [inline src/components/screens/settings/index.js:25]: Please remove all these changes. They don't belong to this PR.

File: src/components/screens/settings/settings.js [hunk 2]
```diff
@@ -13,6 +13,7 @@ import links from '../../../constants/externalLinks';
 import settingsConst from '../../../constants/settings';
 import transactionTypes from '../../../constants/transactionTypes';
 import Dialog from '../../toolbox/dialog/dialog';
+import { Input } from '../../toolbox/inputs';
```
Comment (reyraa) [inline src/components/screens/settings/settings.js:16]: Also this input and the corresponding changes.

File: src/components/shared/transactionPriority/transactionPriority.css [hunk 3]
```diff
@@ -58,6 +58,10 @@
         padding: 5px var(--horizontal-padding-m);
         outline: none;
 
+        &:disabled {
+          background-color: var(--color-platinum);
+        }
+
```
Comment (reyraa) [inline src/components/shared/transactionPriority/transactionPriority.css:64]: And this style

File: src/components/shared/transactionPriority/transactionPriority.js [hunk 4]
```diff
@@ -132,6 +136,7 @@ const TransactionPriority = ({
               className={`${styles.priorityTitle} ${index === selectedPriority ? styles.priorityTitleSelected : ''} option-${priority.title}`}
               onClick={onClickPriority}
               value={index}
+              disabled={index === 0 ? false : shouldDisableOptions}
```
Comment (reyraa) [inline src/components/shared/transactionPriority/transactionPriority.js:139]: I'd put `disabled={index && !priority.value}` and get rid if `shouldDisableOptions `.

File: src/utils/api/lsk/transactions.js [hunk 5]
```diff
@@ -167,18 +168,15 @@ export const broadcast = (transaction, network) => new Promise(
...
-
+export const getTransactionBaseFees = network => liskService.getTransactionBaseFees(network)
+  .then((response) => {
+    const { feeEstimatePerByte } = response.data;
+    return {
+      Low: feeEstimatePerByte.low,
+      Medium: feeEstimatePerByte.medium,
+      High: feeEstimatePerByte.high,
+    };
+  });
```
Comment (reyraa) [inline src/utils/api/lsk/transactions.js:179]: This doesn't account for the catch. We decided if this endpoint throws we should show Low = minFee and keep custom fee input enabled.

File: src/components/shared/transactionPriority/transactionPriority.js [hunk 6]
```diff
@@ -60,7 +60,11 @@ const TransactionPriority = ({
   const [showEditIcon, setShowEditIcon] = useState(false);
   const [inputValue, setInputValue] = useState(undefined);
   const isCustom = selectedPriority === CUSTOM_FEE_INDEX;
-  const isLoading = priorityOptions[0].value === 0;
+  const isLoading = false;
```
Comment (reyraa) [inline src/components/shared/transactionPriority/transactionPriority.js:63]: How do you use `isLoading` then?

**Review by reyraa — CHANGES_REQUESTED** [review]

File: src/components/shared/transactionPriority/transactionPriority.css (hunk 3 above)
Comment (isalga) [reply src/components/shared/transactionPriority/transactionPriority.css:64]: How then we show visually that the priority option is disabled?

Comment (reyraa) [reply src/components/shared/transactionPriority/transactionPriority.css:64]: Ah sorry. My bad

File: src/utils/api/lsk/transactions.js (hunk 5 above)
Comment (isalga) [reply src/utils/api/lsk/transactions.js:179]: @reyraa is handled at `useTransactionPriorities` using `prioritiesLoadError`

File: src/components/shared/transactionPriority/transactionPriority.js [hunk 7]
```diff
@@ -126,16 +127,25 @@ const TransactionPriority = ({
...
-              {priority.title}
-            </button>
-          ))}
+          {tokenRelevantPriorities.map((priority, index) => {
+            let disabled = false;
+            if (index === 3) {
+              disabled = !priority.value && !loadError;
+            } else if (index !== 0) {
+              disabled = !priority.value || loadError;
+            }
```
Comment (UsamaHameed) [inline src/components/shared/transactionPriority/transactionPriority.js:136]: I don't understand the above logic. Could you please briefly explain this? 😃

Comment (isalga) [reply src/components/shared/transactionPriority/transactionPriority.js:136]: @UsamaHameed I updated this so it it much readable. Is it more understandable now or do you still want and explaination?

**Review by UsamaHameed — APPROVED** [review]

**Review by reyraa — APPROVED** [review]
Looks good. Thank you
